packages/destination-actions/src/destinations/fullstory/request-params.ts

```diff
@@ -58,6 +109,7 @@ export const setUserPropertiesRequestParams = (settings: Settings, userId: ID, r
     options: {
       ...defaultParams.options,
       method: 'post',
+      // TODO(nate): Specify json instead of body which will JSON.stringify for us and set the correct content-type header
```

Comment:
I'm planning to open a fast follow PR that cleans up artifacts like this from earlier iterations. Omitted here to keep PR surface smaller and more focused on the new `trackEvent` action.